### ISSUE TITLE
Content Switcher component not AVT compliant

### ISSUE THREAD

**Issue by chas-moseley** [carbon-design-system/carbon#5696]
## Environment

> Operating system - All

> Browser - All

> Automated testing tool and ruleset - DAP (Dynamic Assessment Plugin) was used to discover ruleset 1146 violation.

> Assistive technology used to verify - none

## Detailed description

> What version of the Carbon Design System are you using? - 10.9.3

> What did you expect to happen? Clean report in the area of the Carbon components

> What happened instead? Issues were reported

> What WCAG 2.1 checkpoint does the issue violate? - G108

## Steps to reproduce the issue

1. Create a blank web page that passes AVT specifications.
2. Place a Content Switcher component on the web page.
3. Perform an AVT verification.
4. Expect no violations.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/content-switcher-enhanced-vpmez

## Additional information
- Notes
An additional "enhanced" version of the ContentSwitcher was included in the sandbox that demonstrates a method to make the ContentSwitcher be more compliant.

**Comment by dakahn** [comment]
I'm unable to recreate this issue with the latest Accessibility Checker (DAP) ruleset. Closing. I'll reopen if there's something I missed.
